docs/adrs/notifications.md

```diff
@@ -0,0 +1,18 @@
+# Notifications
+
+## Context
+
+superwerker creates an OpsItem in Systems Manager OpsCenter for emails received by the RootMail feature. Without notifications for new OpsItems, users need to manually check the OpsCenter for new items.
+
+## Decision
+
+- Use CloudWatch Events to trigger an AWS Lambda function whenever a new OpsItem is created.
+- Publish a message to an SNS topic for every new OpsItem.
+- Use native email subscriptions for SNS to notify a specified email address about new messages.
+- If no email address is provided, no SNS topic is created.
+
+## Consequences
+
+- Users can provide an email address when creating the superwerker CloudFormation stack.
```

Comment:
We decided against using the root email address. But I don't recall why. Did we scope it out because we wanted to keep the MVP small? Nevertheless, we should document the decision if it was on purpose.

Reply:
> We decided against (re-)using the existing RootMail address because critical notifications about an AWS account should not land in an mailbox in the same AWS account.